lib/cc/engine/bundler_audit.rb

```diff
@@ -12,3 +13,16 @@ module BundlerAudit
     end
   end
 end
+
+# Patch Bundler::Audit::Scanner to prevent network access during insecure
+# source checks
+
+module Bundler
+  module Audit
+    class Scanner
+      def internal_host?(_uri)
+        false
```

Comment:
That sounds like we'll be generating false positives. Maybe that's ok because they can be excluded later?